extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -215,6 +216,13 @@ export class AmpAutocomplete extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    const doc = this.element.ownerDocument;
+    const isEmail = doc && isAmp4Email(doc);
+    userAssert(
+      !isEmail || doc.documentElement.hasAttribute('enable-amp-autocomplete'),
```

Comment:
The PR comment mentions `amp-autocomplete-opt-in`, while this specifies `enable-amp-autocomplete`.

Can this attribute begin with `data-` instead, so as to not have collisions with html properties?

Reply:
Sorry for the confusion -- hadn't pushed the commit renaming `enable-amp-autocomplete` to `amp-autocomplete-opt-in`.

IIUC all `data-` attributes are allowed and we explicitly want to support a disallowed document flag so email providers can inject it on their own to enable the feature.